core/templates/external-js/core-url.tid

```diff
@@ -0,0 +1,3 @@
+title: $:/core/save/offline-external-js/coreURL
+
+tiddlywikicore-5.2.3.js
```

Comment:
The version should not be hard coded here, but I don't know how to make that happen.